src/app.js

```diff
@@ -74,18 +64,29 @@ app.post('/', async (req, res) => {
       data: result,
     })
   } else {
-    const callRes = await fetch(EGRESS_URLS, {
-      method: 'POST',
-      headers: {
-        'Content-Type': 'application/json',
-      },
-      body: JSON.stringify({
-        data: result,
-      }),
-    })
-    if (!callRes.ok) {
-      return res.status(500).json({ status: false, message: `Error passing response data to ${EGRESS_URLS}` })
+    const urls = []
+    const eUrls = EGRESS_URLS.replace(/ /g, '')
+    if (eUrls.indexOf(',') !== -1) {
+      urls.push(...eUrls.split(','))
+    } else {
+      urls.push(eUrls)
     }
+    urls.forEach(async url => {
+      if (url) {
+        const callRes = await fetch(url, {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({
+            data: result,
+          }),
+        })
+        if (!callRes.ok) {
+          console.error(`Error passing response data to ${url}`)
+        }
+      }
+    })
```

Comment:
I would put it in a separate function like in https://github.com/weeve-modules/opcua-subscribe. Other than that, please see my comments in https://github.com/weeve-modules/opcua-subscribe

Reply:
doesn't have to be since it's only used here.

Reply:
still, it's a whole separate functionality that can be separated under one topic "send data further". separating it also improves modularity in case we need it later

Reply:
separated.